src/Lecture3.hs

```diff
@@ -137,11 +156,14 @@ data List1 a = List1 a [a]
 
 -- | This should be list append.
 instance Semigroup (List1 a) where
-
+  (<>) :: List1 a -> List1 a -> List1 a
+  List1 x xs <> List1 y ys = List1 x (xs ++ (y : ys))
 
 {- | Does 'List1' have the 'Monoid' instance? If no then why?
 
 instance Monoid (List1 a) where
+  The 'List1' has no the 'Monoid' instance because it always contains at least one element. 
+  After concatenating the list with the element, we get a different list.
```

Comment:
That's correct! 👍🏻 
Or, in other words, there's no such thing as "empty non-empty list" 🙂